src/utils/openvpn.ts

```diff
@@ -315,54 +315,51 @@ export class VpnManager extends EventEmitter {
 		});
 	}
 
-	public start() {
-		return Bluebird.resolve(this.killOrphan())
-			.tap(() => {
-				this.process = spawn('/usr/sbin/openvpn', this.args(), {
-					stdio: ['ignore', 'pipe', 'pipe'],
-				});
-				// proxy logs from the child process stdout/stderr
-				this.process.stdout!.pipe(es.split()).on('data', data => {
-					this.emit('log', VpnLogLevels.n, data);
-				});
-				this.process.stderr!.pipe(es.split()).on('data', data => {
-					this.emit('log', VpnLogLevels.n, data);
-				});
-				// proxy error events from the child process
-				this.process.on('error', err => {
-					this.emit('process:error', err);
-				});
-				this.process.on('exit', (code, signal) => {
-					this.emit('process:exit', code, signal);
-				});
-			})
-			.then(() => this.waitForStart());
+	public async start() {
+		await this.killOrphan();
+		this.process = spawn('/usr/sbin/openvpn', this.args(), {
+			stdio: ['ignore', 'pipe', 'pipe'],
+		});
+		// proxy logs from the child process stdout/stderr
+		this.process.stdout!.pipe(es.split()).on('data', data => {
+			this.emit('log', VpnLogLevels.n, data);
+		});
+		this.process.stderr!.pipe(es.split()).on('data', data => {
+			this.emit('log', VpnLogLevels.n, data);
+		});
+		// proxy error events from the child process
+		this.process.on('error', err => {
+			this.emit('process:error', err);
+		});
+		this.process.on('exit', (code, signal) => {
+			this.emit('process:exit', code, signal);
+		});
+		return await this.waitForStart();
```

Comment:
Since `this.waitForStart()` returns void this would be slightly clearer as
```suggestion
		await this.waitForStart();
```
to avoid the implication that the `waitForStart` return value is meaningful